saleor/graphql/order/tests/test_order.py

```diff
@@ -3597,6 +3630,7 @@ def test_invalid_order_when_updating_lines(
     content = get_graphql_content(response)
     data = content["data"]["orderLineUpdate"]
     assert data["errors"]
+    assert not order_update_webhook_mock.called
```

Comment:
I see, that in code, we mostly use `assert_not_called()` so it will be nice to unify this. But I also see that is some places where we check this in the way you did it. So I don't press for change, it's up to you.